components/src/atoms/list-item/ListItem.html

```diff
@@ -0,0 +1,7 @@
+<component
```

Comment:
I'm thinking of moving this `ListItem` component to `components/src/atoms/list/item` and update imports in List stories, what do you think?

Reply:
https://github.com/SnowdogApps/alpaca-storybook/pull/156#discussion_r274013594